testsuite/httpx/__init__.py

```diff
@@ -27,10 +27,10 @@ def __init__(self, msg, response):
 
 class HttpxBackoffClient(Client):
     """Httpx client which retries unstable requests"""
-    RETRY_CODES = {503}
 
     def __init__(self, *, verify: Union[Certificate, bool] = True, cert: Certificate = None, **kwargs):
```

Comment:
This is fine, but how about adding the retry_codes as an optional init parameter?

Reply:
With this implementation u can easily add a new retry code in your test: `client.add_retry_code(404) and you don't have to define a new client fixture.